app/views/dashboard/_index_partials/_metadata_download.html.erb

```diff
@@ -0,0 +1,7 @@
+<div class="panel panel-default">
+  <div class="panel-body">
+    <div class="col-xs-6 col-sm-12 col-md-6 user-info">
+      <%= link_to "Admin metadata download", "/dashboard/admin_metadata_download" %>
```

Comment:
Better to put the actual path as the second parameter to "link_to" helper function